src/script.sh

```
@@ -20,8 +20,58 @@ main() {
+
+    # Compress and index if needed
+    for var in vcf_path gvcf_path; do
+        path="${!var}"
+        if [[ "$path" != *.vcf.gz ]]; then
+            bgzip -c "$path" > "${path}.gz"
+            printf -v "$var" '%s' "${path}.gz"
+        fi
+        bcftools index -t "${!var}"
+    done
+
+    if [[ min_qual -gt 0 ]]; then
```

Shellcheck (0.10.0)

[warning] min_qual is referenced but not assigned.

(SC2154)